adapter.go

```diff
@@ -128,6 +128,8 @@ func (a *Adapter) open() {
 
 		if a.driverName == "postgres" {
 			db, err = gorm.Open(a.driverName, a.dataSourceName+" dbname=casbin")
+		} else if a.driverName == "sqllite3 {
```

Comment:
You spelled wrong ``sqlite3``. Moreover, you code is missing a `"`. Did you ever run and test you code when submitting the PR? If not, I suggest so.

And to guarantee it, you can write a test case and create a repo (or branch, whatever). I want to see the test case works in my local machine. But the test case will not end up in this master branch, as it imports sqlite dependency.

Reply:
I did not pull the repo down for this "simple" fix, felt that it was a little overkill to setup the project locally etc. 

I apologise for the typo, githubs editor is not a full IDE which marks syntax errors 😅.

> Did you ever run and test you code when submitting the PR? If not, I suggest so.

yes of course, this project as a vendored dependency (with dep). So i changed the source code in of my vendored version. As dep removes all non go files, like the .git folder, i could not just fork this project, add my fork as remote and push my changes. I had to either fork, pull and set up the project. Or just simply open the gihub editor and make the changes. 

My mistake came that i manually re-wrote my changes instead of copy pasting it, hence these silly mistakes.

> And to guarantee it, you can write a test case and create a repo (or branch, whatever). I want to see the test case works in my local machine. But the test case will not end up in this master branch, as it imports sqlite dependency.

I dont understand how you want me to write a test case that is not going to be included in master because it adds a dependency. I would suggest either come up with a structural fix for this, either .tar the test files and untar  then in CI and actually run tests. Moreover, go modules and go get do not import test only dependencies. So i dont really see the issue here. 

https://github.com/golang/go/issues/26913#issuecomment-411976222 

> The -t flag instructs get to also download the packages required to build the tests for the specified packages.
https://golang.org/cmd/go/#hdr-Download_and_install_packages_and_dependencies

Reply:
OK. Then never mind the test case thing. I don't want things to be too complicated either. You can copy-and-paste the fix from your local working code to this PR.

And please merge your two commits into one.

Reply:
Ideally, the PR only contains one commit, and this commit shows the best state of what you are trying to do. It's not very good for the PR author to create a faulty commit, then use another commit to fix the previous commit inside the PR. The commit history of this repo doesn't need to show this error-and-fix process from 3rd-party.